build-plugins/kmp-conventions/src/main/kotlin/aws/sdk/kotlin/gradle/kmp/ConfigureTargets.kt

```diff
@@ -112,22 +108,15 @@ fun Project.configureKmpTargets() {
         // FIXME Configure JS
 
         if (NATIVE_ENABLED) {
-            if (hasApple) {
+            if ((hasApple || hasDesktop) && HostManager.hostIsMac) {
```

Comment:
Was the `hasDesktop` addition necessary here? IIRC we don't use the "desktop" source set anywhere. Was this just for correctness?

Reply:
We can leave it in place for now and look at simplifying after development is complete